index.js

```diff
@@ -22,7 +24,8 @@ const schema = buildSchema(`
     }
 
     type Mutation {
-        shortenUrl(url: String!): String
+        shortenUrl(url: String!): String,
+        originalUrl(id: String!): String
```

Comment:
@OlusanyaJegs You can return the originalUrl in the shortenUrl mutation. This means it'll be an object and not a String anymore.

Reply:
I tried to do that before and i was getting the error message below

"Error: Unknown type "originalUrl".